src/playlist-controller.js

```diff
@@ -192,6 +192,7 @@ export class PlaylistController extends videojs.EventTarget {
     this.playlistExclusionDuration = playlistExclusionDuration;
     this.maxPlaylistRetries = maxPlaylistRetries;
     this.enableLowInitialPlaylist = enableLowInitialPlaylist;
+    this.hasManagedMediaSource_ = false;
```

Comment:
nit: To save a few lines of code, you could change this to:
```
this.hasManagedMediaSource_ = experimentalUseMMS && window.ManagedMediaSource;
```

Reply:
The intent of the flag was to signal that the managedMediaSource has been created and currently exists. If we set it here we should rename the flag to `useManagedMediaSource_` or `createManagedMediaSource_`, as it hasn't been created yet.